PathOfBuildingCommunity/PathOfBuilding
# Support URLs in the Notes Tab

Fixes allowing URLs in Notes.

### Description of the problem being solved:
Allows a way for build creators to supply URLs to the atlas tree in their builds, offering an alternative to having to support the Atlas Tree.

### Steps taken to verify a working solution:
- Open a build, type some text in the Notes tab.
1. Double Click some text and observe the browser is not opened.
- Type or paste in a URL.
EG: http://reaper.fm
- Double Click the URL and observe the browser opens with the url.
- Type or paste in c:\Windows\notepad.exe .
- Double Click the notepad text and observe the browser is not opened, nor does notepad.


### Some Notes that showcases this PR:
<pre>Notes

Stuff

Link https://tinyurl.com/yp22t3tu
Link http://reaper.fm
Link https://tinyurl.com/yp22t3tu3
Link https://www.lua.org/pil/20.1.html
Link https://www.google.com/search?q=path+of+exile+righteous+fire+supports&client=firefox-b-d&sxsrf=ALiCzsY9NqKBUO9_12uCH8HAWrOyOv6QsA%3A1654593067138&ei=KxafYuqJCLH04-EPrJGxgAg&oq=path+of+Exile+&sclient=gws-wiz
Executable c:\Windows\notepad.exe

More Stuff      

/ ? ://?
.
https://www.youtube.com/watch?v=fpELYXYTGeY</pre>

### New descriptive text
![image](https://user-images.githubusercontent.com/4302241/172357034-00639ae1-449b-492e-b8b7-779102187b6e.png)

edit: 31/05/2022
Merge current DEV and solve merge conflicts
